uiowa/uiowa
# Allow the sitenow request form login to pass query string parameters

Adjusts the logic that adds a login link to the SiteNow request form to allow and preserve variables that are passed to the page.

# How to test

```
ddev blt ds --site sitenow.uiowa.edu
```
- Visit https://sitenow.uiowa.ddev.site/request-site?domain=test.uiowa.edu&requestor=sadamshiett
- Confirm that when you hover over the login link, you see the query string parameters are transferred to it
- Confirm that adding, removing, and changing the parameters adjusts the login link
- Please note that I was not able to figure out how to avoid overwriting the `destination` param with one if it's provided.
